soc/nxp/imxrt/imxrt6xx/Kconfig.defconfig

```diff
@@ -115,3 +115,32 @@ choice USB_MCUX_CONTROLLER_TYPE
 endchoice
 
 endif # SOC_MIMXRT685S_CM33
+
+if SOC_MIMXRT685S_ADSP
+
+config SOC_TOOLCHAIN_NAME
+	string
+	default "nxp_rt600_adsp"
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 198000000
+
+config XTENSA_CCOUNT_HZ
+	default SYS_CLOCK_HW_CYCLES_PER_SEC
+
+config SYS_CLOCK_TICKS_PER_SEC
+	default 1000
+
+config CACHE
```

Comment:
What's this for? I wasn't able to find `CONFIG_CACHE` anywhere?

Reply:
```
At drivers/cache/Kconfig:4
Included via /home/nxf25322/work/sof_dir3/zephyr/Kconfig:8 -> Kconfig.zephyr:48 -> drivers/Kconfig:14
Menu path: (Top) -> Device Drivers

  menuconfig CACHE
        bool "External cache controller drivers"
        default y if CACHE_MANAGEMENT(=y)
        help
          Enable support for external cache controllers drivers
```

I'm not sure what is the default value of a config symbol if not specified.